src/organisms/cards/PolicyCard/Compare.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Text from 'atoms/Text';
+import Hide from 'wrappers/Hide';
+import Spacer from 'atoms/Spacer';
+import CheckBoxField from 'molecules/formfields/CheckBoxField';
+import styles from './policy_card.module.scss';
+
+export const Compare = ({ compareSelected, onCompare }) =>
+  <Hide hideOn='small' className={styles['compare']}>
+    <Text
+      color='neutral-3'
+      light
```

Comment:
@cisacke will this always stay `light`, even when selected?

Reply:
@jmcolella good q, not sure - I'll ask Jim